src/util/useCheckCanEdit.ts

```diff
@@ -4,12 +4,16 @@ import { Acl } from "@madie/madie-models/dist/Measure";
 const useCheckUserCanEdit = (
   createdBy: string,
   acls: Array<Acl>,
-  draft: boolean = true
+  draft: boolean = true,
+  editTestsOnVersionedMeasures?: boolean
 ): boolean => {
   const { getUserName } = useOktaTokens();
   const userName = getUserName();
   // versioned measures/libraries are always uneditable.
   if (!draft) {
+    if (editTestsOnVersionedMeasures) {
```

Comment:
can I edit if I'm not a owner/sharedwith user?

Reply:
thank you, I missed that scenario I have fixed it now